internal/validation/validation.go

```diff
@@ -0,0 +1,26 @@
+package validation
```

Comment:
Would it make sense to have a JSON Schema here instead? We want to provide one for access rules anyways!

Reply:
@aeneasr Good point! How about we create a follow-up task and get back to it later? We'd like to focus on charts now if you don't mind!